setup.cfg

```diff
@@ -1,6 +1,6 @@
 [versioneer]
 VCS = git
-style = pep440
+style = pep440-post
```

Comment:
What does this change compared so our current configuration? Does it _always_ include the commit in the version? The corresponding PRs in databroker and bluesky didn't have this change, so if we do this we should go back and do it consistently there too.

Reply:
For the other PRs I let versioneer do it's thing.  @tacaswell did you add this explicitly.

Reply:
https://github.com/warner/python-versioneer/blob/master/details.md#how-do-i-select-a-version-style

This is `TAG.postDISTANCE+gHASH`, the default is `TAG+DISTANCE.gHASH`

I have had strong opinions about this in the past but do not recall why...

Reply:
I don't recall either. Can you pull the `-post` into a separate commit so we can keep the projects consistent until you decide?